include/ppx/command_line_parser.h

```diff
@@ -40,6 +40,7 @@ struct StandardOptions
     int                 gpu_index          = -1;
     std::pair<int, int> resolution         = {-1, -1};
     int                 frame_count        = -1;
+    int                 run_time_ms        = -1;
```

Comment:
Could you add it to the usage message below? For both xr and non-xr

Reply:
Ah yep, right.